plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/WorkspacesListItem.tsx

```diff
@@ -226,6 +230,37 @@ export const WorkspacesListItem = ({
   );
 };
 
+const deletingStatuses: readonly WorkspaceStatus[] = ['deleting', 'deleted'];
```

Comment:
Should `deleted` be moved to `offline`? Worried folks might assume a workspace is stuck in a transient state.

Reply:
Actually, that's a good call. I think I over-complicated things, and we can keep the status a binary offline/online
Will fix really quick

Reply:
I just realized how unlikely we are to show deleted workspaces in a filtered list view.

Reply:
Well, I think it still makes sense to make the change – at the very least, the deleting workspaces would show up for "owner:me", which is the first filter people are going to see

Reply:
@Parkreiner I like deleting! We can definitely keep this addition! I more meant that if a workspace is deleted, it is also offline.

In general, these boundary states (deleting, starting, etc) are useful and down the road I'd love to expose more of them as you've done here.